### ISSUE TITLE
Code-insiders AMD Renior GPU acceleration failed on Debian

### ISSUE THREAD

**Issue by YuruiHong** [microsoft/vscode#195688]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.84.0-insider 0c36b92c82064882a228487040187cfc13669c0f x64

- OS Version: Debian 6.5.6-1 (2023-10-07) x86_64 GNU/Linux

Steps to Reproduce:

1. Launch VS Code with `code-insiders --disable-extensions` 
2. The launched window is in color blocks:

![image](https://github.com/microsoft/vscode/assets/83064277/f8d95950-4d85-4f03-8cb7-b8eedb301374)

4. The terminal recursively output the following error:

```txt
Errors:
link failed but did not provide an info log
[12104:1016/190254.756595:WARNING:angle_platform_impl.cc(48)] ProgramGL.cpp:989 (checkLinkStatus): Program link or binary loading failed with no info log.
[12104:1016/190254.756854:ERROR:shared_context_state.cc(81)] Skia shader compilation error
------------------------
// Vertex SKSL
#extension GL_NV_shader_noperspective_interpolation: require
uniform float4 sk_RTAdjust;in float2 position;in float2 localCoord;noperspective out float2 vlocalCoord_S0;void main() {// Primitive Processor QuadPerEdgeAAGeometryProcessor
vlocalCoord_S0 = localCoord;sk_Position = position.xy01;}
// Fragment SKSL
#extension GL_NV_shader_noperspective_interpolation: require
uniform sampler2D uTextureSampler_0_S0;
noperspective in float2 vlocalCoord_S0;void main() {// Stage 0, QuadPerEdgeAAGeometryProcessor
half4 outputColor_S0 = half4(1);float2 texCoord;texCoord = vlocalCoord_S0;outputColor_S0 = (blend_modulate(sample(uTextureSampler_0_S0, texCoord), half4(1)));const half4 outputCoverage_S0 = half4(1);{ // Xfer Processor: Porter Duff
sk_FragColor = outputColor_S0 * outputCoverage_S0;}}
// Vertex GLSL
#version 300 es

#extension GL_NV_shader_noperspective_interpolation : require
precision mediump float;
precision mediump sampler2D;
uniform highp vec4 sk_RTAdjust;
in highp vec2 position;
in highp vec2 localCoord;
noperspective out highp vec2 vlocalCoord_S0;
void main() {
    vlocalCoord_S0 = localCoord;
    gl_Position = vec4(position, 0.0, 1.0);
    gl_Position = vec4(gl_Position.xy * sk_RTAdjust.xz + gl_Position.ww * sk_RTAdjust.yw, 0.0, gl_Position.w);
}

// Fragment GLSL
#version 300 es

#extension GL_NV_shader_noperspective_interpolation : require
precision mediump float;
precision mediump sampler2D;
out mediump vec4 sk_FragColor;
uniform sampler2D uTextureSampler_0_S0;
noperspective in highp vec2 vlocalCoord_S0;
void main() {
    mediump vec4 outputColor_S0 = vec4(1.0);
    highp vec2 texCoord;
    texCoord = vlocalCoord_S0;
    outputColor_S0 = texture(uTextureSampler_0_S0, texCoord, -0.5);
    {
        sk_FragColor = outputColor_S0;
    }
}
```

3. Running `code-insiders --disable-gpu` launches the app normally.

**Comment by Andythem23** [comment]
I have the same problem

**Comment by YuruiHong** [comment]
Seems fixed in #195502
